addons/account_audit_trail/models/mail_message.py

```diff
@@ -3,18 +3,19 @@
 
 from markupsafe import Markup
 
-from odoo import fields, api, models, _
+from odoo import api, fields, models, _
 from odoo.exceptions import UserError
 
 
 class Message(models.Model):
     _inherit = 'mail.message'
 
-    l10n_in_audit_log_preview = fields.Html(string="Description", compute="_compute_l10n_in_audit_log_preview")
-    l10n_in_audit_log_account_move_id = fields.Many2one('account.move', string="Journal Entry", compute="_compute_l10n_in_audit_log_document_name", search="_search_l10n_in_audit_log_document_name")
+    account_audit_log_preview = fields.Html(string="Description", compute="_compute_account_audit_log_preview")
+    account_audit_log_account_move_id = fields.Many2one('account.move', string="Journal Entry", compute="_compute_account_audit_log_document", search="_search_account_audit_log_document_name")
```

Comment:
A bit redundant to repeat `account`
Make the `_compute` and `_search` functions match the field name
```suggestion
    account_audit_log_move_id = fields.Many2one(
        comodel_name='account.move',
        string="Journal Entry", 
        compute="_compute_account_audit_log_move_id",
        search="_search_account_audit_log_move_id",
    )
```